desktop/libs/indexer/src/indexer/templates/importer.mako

```diff
@@ -2385,7 +2385,16 @@ ${ commonheader(_("Importer"), "indexer", user, request, "60px") | n,unicode }
                 name += '.' + wizard.source.streamObject();
               }
             } else if (wizard.source.path()) {
-              name += '.' + wizard.source.path().split('/').pop().split('.')[0];
+              const source_path = wizard.source.path();
```

Comment:
Maybe instead bod this big Logic we should just default the DB name to empty if not match for it?

Reply:
we need to handle the '.' also.
And this condition fulfilling all the cases, either phoenix empty DB or named DB. And also other dialects' DB